src/js/views/patients/sidebar/action/action-sidebar.hbs

```diff
@@ -17,6 +17,7 @@
     <div class="flex u-margin--t-8"><h4 class="action-sidebar__label u-margin--t-8">{{ @intl.patients.sidebar.action.actionSidebarTemplate.ownerLabel }}</h4><div class="flex-grow" data-owner-region></div></div>
     <div class="flex u-margin--t-8"><h4 class="action-sidebar__label u-margin--t-8">{{ @intl.patients.sidebar.action.actionSidebarTemplate.dueDateLabel }}</h4><div class="flex-grow" data-due-region></div></div>
     <div class="flex u-margin--t-8"><h4 class="action-sidebar__label u-margin--t-8">{{ @intl.patients.sidebar.action.actionSidebarTemplate.durationLabel }}</h4><div class="flex-grow" data-duration-region></div></div>
+    {{#if form}}<div class="flex u-margin--t-8"><h4 class="action-sidebar__label u-margin--t-8">{{ @intl.patients.sidebar.action.actionSidebarTemplate.attachmentLabel }}</h4>{{/if}}<div class="flex-grow" data-attachment-region></div></div>
```

Comment:
this is leaving a trailing `</div>` we'll really need to wrap the whole thing.  Should also probably be `hasForm` or `hasAttachment`